components/Contact.js

```diff
@@ -0,0 +1,22 @@
+var Contact = React.createClass({
+  propTypes: {
+    item: React.PropTypes.object.isRequired,
+  },
+
+  render: function () {
+    return (
+      <div className={'contactItem'}>
```

Comment:
Tak wiem, że było tak w przykładzie w module, ale nie ma potrzeby używać nawiasów klamrowych `{}` jednocześnie z apostrofom `'`. Nawiasy klamrowe służą do osadzenia wyrażenia JS które zwraca jakąś wartość, w Twoim przypadku masz stałą wartość jako łańcuch znaków, wystarczy zapisać `<div className='contectItem'>. To się tyczy wszystkich atrybutów zapisanych w ten sposób. Nie musisz tego zmieniać od razu w tym zadaniu, ale wiedz, że nie ma takiej potrzeby aby pisać w ten sposób, kod będzie czytelniejszy i łatwiejszy do napisania bez tej konstrukcji.